client/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+members = [
+    "hwctl",
+    "hwlib"
+]
+resolver = "2"
+exclude = ["rust-vendor",]
+
+[workspace.metadata.vendor-filter]
+platforms = ["x86_64-unknown-linux-gnu", "aarch64-unknown-linux-gnu"]
```

Comment:
Do we certify any other architectures where this this could be a problem?

Reply:
currently, the library supports amd64 architecture and has limited support for arm64. that's why we pass both these architectures here, but pack only for amd64 since this is the only architecture the client is ready to be used on.

Reply:
Ack